### PR TITLE
README: add a note about the VM template

### PR DESCRIPTION
**Pull request by goneri**
Depends-On: https://github.com/ansible-collections/vmware.vmware_rest/pull/331

### CODE REVIEW

**Review by jillr — APPROVED** [review]
Documentation LGTM when CI is passing

**Comment by mariolenz** [comment]
@goneri I think the gate pipeline for your PR will succeed as soon as #331 is merged. The `milestone` branch has been advanced from somewhere in February to last Monday recently, and there have been some additional sanity checks added in the meantime. #331 should fix them.

cc @jillr

**Pull request merged**
